src/commons/controlBar/ControlBarShareButton.tsx

```diff
@@ -20,11 +20,13 @@ type StateProps = {
   shortURL?: string;
   key: string;
   isSicp?: boolean;
+  programConfig: object;
```

Comment:
What is this `programConfig` object? Can this be typed better?

Reply:
Already changed to `Partial<ShareLinkState>`